### ISSUE TITLE
DB/Quest: Massparsed 9.0.2 quest data

### ISSUE THREAD

**Issue by mdX7** [TrinityCore/TrinityCore#27333]
### Description

slow query

### SQL fix

```sql
https://gist.githubusercontent.com/mdX7/9b18cb7523bced853b6e5f7eac25e733/raw/bc931caf5879931da23dc281b62430edd040aaad/902.sql
```

### Branch

master

### TC rev. hash/commit

aeece162b9d93c9d248ea8ba464cf28ae532f034

**Comment by TQPS** [comment]
Is this ready to merge?

**Comment by TQPS** [comment]
This can be closed now? Done in [8d83c78](https://github.com/TrinityCore/TrinityCore/commit/8d83c78618d5cde9f09e68e193ecb366b52b8381)

**Comment by Naddley** [comment]
yes :D https://github.com/TrinityCore/TrinityCore/commit/78f8acb4090206c9313a9e9749c6938a0284b91b
